### ISSUE TITLE
Not the same coordinate system?

### ISSUE THREAD

**Issue by xiangyuanhang** [facebookresearch/habitat-api#337]
## ❓ Questions and Help
Hi,
I'm trying to select some start/end points manully from Gibson-dataset.
First I opened the glb model using blender.
And then I visualized several points  using the coords in the point-nav json file.
However, these points appear not in the house. The picture below shows the model Adrian.glb and several points from  pointnav/train/content/Adrian.json
![image](https://user-images.githubusercontent.com/33518874/77184321-9b71e880-6b0a-11ea-8eed-ca9dd8f0da51.png)

So I wonder the points in the json file  are not in the same coordinate system with the glb model?
If so, how could I get the correct point coords for use in json by choosing manually from the scene?

Thank you so much.

**Comment by erikwijmans** [comment]
Yes.  gibson (and mp3d) are -Z gravity meshes.  Habitat rotates these to -Y.  See here: https://github.com/facebookresearch/habitat-sim/issues/524

**Comment by xiangyuanhang** [comment]
Thanks for replying!
I have tried the method you provided. Picture below shows what I have done. The mesh is Adrian from Gibson. Basically I choose a goal point from Adrian.json(pointnav) and then I transform the position of the point using the method you mentioned. Then I visualize the point in blender. However, the point is still outside of the house...
![image](https://user-images.githubusercontent.com/33518874/77245424-7770ed00-6c59-11ea-969c-0e4644095269.png)
![image](https://user-images.githubusercontent.com/33518874/77245458-b0a95d00-6c59-11ea-8714-2a3efc9dfe78.png)
Is it possible that the method you provide is only for mp3d but not for Gibson?
Thank you so much.

**Comment by erikwijmans** [comment]
No, it's the same for Gibson.  IIRC, there is also something weird Blender does with the coordinate system.  CC @aclegg3
